content/terraform-docs-common/docs/cloud-docs/api-docs/workspaces.mdx

```diff
@@ -141,6 +141,7 @@ By supplying the necessary attributes under a `vcs-repository` object, you can c
 | `data.attributes.vcs-repo` | object  | none | Settings for the workspace's VCS repository. If omitted, the workspace is created without a VCS repo. If included, you must specify at least the `oauth-token-id` and `identifier` keys. |
 | `data.attributes.working-directory`             | string  | (nothing) | A relative path that Terraform will execute within. This defaults to the root of your repository and is typically set to a subdirectory matching the environment when multiple environments exist within the same repository. |
 | `data.attributes.setting-overwrites` | object | none | The keys in this object are attributes that have project-level defaults. Each attribute key stores a boolean value which is `false` by default. To overwrite the default inherited value, set an attribute's value to `true`. For example, to set `execution-mode` as the project default, set `setting-overwrites.execution-mode` to `true`. |
+| `data.attributes.hyok-enabled`       | boolean | `false` | Whether this workspace will secure HCP Terraform artifacts using Hold Your Own Key (HYOK) encryption. Once enabled, this can not be disabled. |
```

Comment:
```suggestion
| `data.attributes.hyok-enabled`       | boolean | `false` | Whether this workspace will secure HCP Terraform artifacts using Hold Your Own Key (HYOK) encryption. Once enabled, this can not be disabled. HCP Terraform only, requires the [HCP Terraform Premium plan](https://www.hashicorp.com/en/pricing?tab=terraform.)   |
```